crime-applications-adaptor/src/main/java/uk/gov/justice/laa/crime/applications/adaptor/mapper/CrimeApplyMapper.java

```diff
@@ -0,0 +1,112 @@
+package uk.gov.justice.laa.crime.applications.adaptor.mapper;
+
+import lombok.RequiredArgsConstructor;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.stereotype.Service;
+import uk.gov.justice.laa.crime.applications.adaptor.model.crimeapplicationsadaptor.Address;
+import uk.gov.justice.laa.crime.applications.adaptor.model.crimeapplicationsadaptor.Applicant;
+import uk.gov.justice.laa.crime.applications.adaptor.model.crimeapplicationsadaptor.CrimeApplication;
+import uk.gov.justice.laa.crime.applications.adaptor.model.criminalapplicationsdatastore.MaatApplication;
+import uk.gov.justice.laa.crime.applications.adaptor.model.criminalapplicationsdatastore.general.Provider;
+
+import java.util.List;
+
+/**
+ * The responsibility of this class is to encapsulate the required logic to map from a
+ * Criminal Applications Datastore response
+ * to a
+ * Crime Applications Adaptor MAAT Application (which should be structured like a MAAT ApplicationDTO)
+ */
+@Service
+@RequiredArgsConstructor
```

Comment:
I think @RequiredArgsConstructor is not required , please verify.

Please use @Component instead of @Service as it is not a service class.

Reply:
Good point, have done both as Java by default always provides a no-args constructor (calling super()) if no other constructor is defined.